docs/DeveloperGuide.md

```diff
@@ -398,6 +398,24 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case ends.
 
+**Use case: Add a tag to a person**
+
+**MSS**
+
+1. User requests to list persons
+2. Coach2K22 shows a list of persons
+5. User requests to attach an existing tag to a person
```

Comment:
Some problems with the numbering. Also, should it be a new tag instead?

Reply:
Oops. Fixed